lib/core/regs/consent.test.js

```diff
@@ -15,6 +15,53 @@ const denyAll = {
   measureAdvertisingPerformance: false,
 };
 
+describe("applicableReg", () => {
+  it("returns gdpr when cmp indicates gdpr applies", () => {
+    expect(applicableReg("can", { gdprApplies: true })).toBe("gdpr");
+    expect(applicableReg("us", { gdprApplies: true })).toBe("gdpr");
+    expect(applicableReg(null, { gdprApplies: true })).toBe("gdpr");
+
+    expect(applicableReg("can", { gppSectionIDs: [2] })).toBe("gdpr");
```

Comment:
What happens if you have multiple section IDs. For example, `[2, 5]` or `[2, 999]`. I assume `gdpr` wins in both cases? Can you add test cases for it

Reply:
Updated